client/src/main/java/gui/controllers/GameSceneController.java

```diff
@@ -23,6 +31,8 @@
 
     public void initialize() {
         appInstance = ClientAppRunner.getInstance();
+        String playerName = appInstance.getPlayer().getName();

```

Comment:
you can inline this variable

Reply:
I think it's more clear that way. That was my intention.